src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -11,7 +11,7 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
 
   @Deprecated
   void loadVaultSecrets(List<Map<String, Object>> vaultSecrets) {
-    this.vaultSecrets = ['unknown': vaultSecrets]
+    config.vaultSecrets = ['unknown': vaultSecrets]
```

Comment:
we should look too remove this at some point i guess, don't want to maintain it forever